stable/hazelcast/Chart.yaml

```
@@ -1,6 +1,6 @@
 name: hazelcast
-version: 1.0.1
-appVersion: "3.10.2"
+version: 1.0.2
```

```suggestion
version: 1.1.0
```

since there a bit more changes going on